content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,229 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
```

Comment:
@SoyGema Hi Gema, thanks for your feedback!

This is indeed an early draft and things are still changing quite a bit. But maybe to give more clarity w.r.t. its intentions. The getting started guide on model management is not supposed to replace existing documentation but to supplement it.

In general, the documentation has loosely 4 parts:
1. High level concept pages like [this one](https://dvc.org/doc/use-cases/model-registry)  where the general concepts and motivation behind features are explored. This is for people who want to learn about why they might want to use a specific tool and/or our philosophy behind it.
2. User guides where you can quickly jump to clear steps on how to achieve a specific task - e.g. the one you linked to [here](https://dvc.org/doc/studio/model-registry). This is for people who are already using the tool and are looking for a way to do something in particular with it.
3. A command reference, with detailed explanation of a specific library's commands (in case of the model registry you typically use the studio UI but that is mostly based on the GTO library, so the closest example is [its command reference](https://dvc.org/doc/gto/command-reference)).
4. Getting started guides - the aim of those is not to go into too much detail but to guide a user through an end2end workflow with as little friction as possible. This is for people who are new to the tool and who want to try things out and get a more hands-on sense of what it does and how it does that. We have those for [experiments](https://dvc.org/doc/start/experiments) and [data management](https://dvc.org/doc/start/data-management) already and in this PR we are working on the same for model management.

That said, I think we can perhaps make this structure clearer in our docs sometimes. If you'd like to contribute, maybe the best thing at the moment is to wait for this draft to be finished (a few more weeks from now at most I think) and to review it once it is deployed. We are happy to incorporate any suggestions that will make the docs clearer and easier to use.

Reply:
Hey! Thanks for the quick reply . I really appreciate it . 

Thanks for aknowleding that lack of structure might be an issue for users. Please, think consider adding  length and focus , and empathy regarding time-to-things-working .

My apologies if this seemed as a contribution 
:( , as it wasn't intended to be :( but more providing feedback to try for you to improve faster while being a work in progress. Governance with respect to contribs  is definitely a challenge you are facing as mantainers... I'll try to have a look at good first issues or more canonical ways of contributing !!🦾

Reply:
Cc @shcheklein I'll not contribute more in drafts

Reply:
Hey @tibor-mach just for clarifying and my apologies in advance for not saying this before... I'm tagging Ivan because we've sometimes discuss about how keep contributing on my end, not by any means something wrong... I'm searching for optimal ways and keeping track on this... I just thought this could be seen in another way and I wanted to clarify...

Reply:
@SoyGema No worries at all :-) Like I mentioned, if you'd like to contribute I think a very good way to help us with this particular thing is to test the guide once this pull request is ready for merging or once it is deployed. You could go through the guide and see if it makes sense to you, if you feel it is a good introductory guide and the comments on how that relates to the rest of the documentation are also welcome.

This feedback can be very helpful for us and we might tweak the docs somewhat based on it to make things clearer for users.

Reply:
Thanks for letting me know what´s more useful for you! :)